fastai/vision/augment.py

```diff
@@ -420,14 +453,25 @@ def _prepare_mat(x, mat):
 class AffineCoordTfm(RandTransform):
     "Combine and apply affine and coord transforms"
     order,split_idx = 30,None
-    def __init__(self, aff_fs=None, coord_fs=None, size=None, mode='bilinear', pad_mode=PadMode.Reflection,
-                 mode_mask='nearest', align_corners=None, **kwargs):
+    def __init__(self,
+        aff_fs:(callable,[callable])=None, # Affine transformations function for a batch
```

Comment:
```suggestion
        aff_fs:(callable,list)=None, # Affine transformations function for a batch
```
Just list for ones like these instead of `[callable]`

Reply:
Done. I still have a type like this in this PR. example `draw:(int, [int], callable)` Do you think I need to change [int] to list? I believe the user can be confused to use a list of callable or list of int. 

Thanks

Reply:
@dienhoa yes, because [int] will cause errors for users that use IDEs.

Reply:
I change very [type] to list in the new commit